### ISSUE TITLE
Remove pygalmesh dependency

### ISSUE THREAD

**Issue by stefsmeets** [hpgem/nanomesh#124]
`tetgen` works well with 3d meshes, therefore the dependency on `pygalmesh` integration is no longer needed after #123 is addressed.

- [x] periodic_utils.py
- [x] setup.py
- [x] requirements.txt
- [x] generators.py
- [x] generator notebooks
- [x] test_notebooks.sh
- [x] github actions

**Comment by stefsmeets** [comment]
This issue benefits from:
- [ ] #106
- [ ] #124
